### ISSUE TITLE
Accessibility plot not invalidated when opportunities change or comparison project removed

### ISSUE THREAD

**Issue by abyrd** [conveyal/analysis-ui#1249]
In the single point analysis page, when using worker-side accessibility computations, if you change the destination set the cumulative accessibility plot is no longer reflecting parameters. Following existing convention it should be grayed out (along with some other controls) until the analysis is re-run.

Something similar is true for comparing two requests: when you remove the comparison project (by clicking the X) the plot is not grayed out. If you change the project or scenario rather than clear them, the plot is grayed out as expected.

**Comment by trevorgerhardt** [comment]
Fixed in #1251
